rs/types/types/src/exhaustive.rs

```diff
@@ -284,7 +284,10 @@ impl<K: ExhaustiveSet + std::cmp::Ord> ExhaustiveSet for BTreeSet<K> {
 
 impl ExhaustiveSet for RejectCode {
     fn exhaustive_set<R: RngCore + CryptoRng>(_: &mut R) -> Vec<Self> {
-        RejectCode::iter().collect()
+        RejectCode::iter()
+            // TODO(MR-610): Drop this after `SysUnknown` is supported on mainnet.
+            .filter(|code| *code != RejectCode::SysUnknown)
```

Comment:
I assume there is no safer way to ensure we don't forget to remove this than linking the ticket?

Reply:
I guess introducing a new certification version would help here. But I'm fine with this.

Reply:
That's what the test failure message suggested. I also don't think there's a safer way.

Reply:
[Sorry, only replied to David above, GitHub did not render Christian's comment without refreshing.]

We can definitely add a new certification version, as said above. But it won't help with `ExhaustiveSet` and the CUP compatibility test. Or with remembering to drop this line after a couple of replica releases.